pkg/notifications/slack/builder.go

```diff
@@ -27,10 +34,34 @@ func NewNotifier() (notifications.Notifier, error) {
 		channelID: channelID,
 	}
 
-	// Check that the slack app has been added to the channel in the workspace
+	// Check that the Slack app has been added to the channel in the workspace
 	if _, err := n.client.GetConversationInfo(channelID, false); err != nil {
 		return nil, err
 	}
 
 	return n, nil
 }
+
+func getSlackChannelID() (string, bool) {
+	// Read channel ID from env var
+	channelID, ok := os.LookupEnv(slackChannelID)
+	return channelID, ok
```

Comment:
nite: could be simplified to just `return os.LookupEnv(slackChannelID)`

Reply:
👍 Updated.